book/07-git-tools/sections/revision-selection.asc

```diff
@@ -82,8 +82,10 @@ That's 1,200 times the number of grains of sand on the earth.
 
 Here's an example to give you an idea of what it would take to get a SHA-1 collision.
 If all 6.5 billion humans on Earth were programming, and every second, each one was producing code that was the equivalent of the entire Linux kernel history (6.5 million Git objects) and pushing it into one enormous Git repository, it would take roughly 2 years until that repository contained enough objects to have a 50% probability of a single SHA-1 object collision.
-Thus, a SHA-1 collision is less likely than every member of your programming team being attacked and killed by wolves in unrelated incidents on the same night.
+Thus, an organic SHA-1 collision is less likely than every member of your programming team being attacked and killed by wolves in unrelated incidents on the same night.
 
+If you dedicate several thousands of dollars' worth of computing power to it, it is possible to synthesize two files with the same hash, as proven on https://shattered.io/[] in February 2017.
```

Comment:
```suggestion
If you dedicate several thousands of dollars' worth of computing power to it, it is possible to synthesize two files with the same hash, as proven on https://shattered.io/[] on February 2017.
```

I think you meant `on` here?

Reply:
No. In English, events happen *in* months, not *on* them.

Reply:
Oh wait, you're right, and I'm wrong, `in` is right in this case.... :+1: